### PR TITLE
Failed to load challenges text in index.html

### PR DESCRIPTION
**Pull request by Tobias-hubs**
image

### CODE REVIEW

File: challenges.js [hunk 1]
```diff
@@ -9,23 +9,40 @@ const challengeSort = {
...
 			}
 		}
-			
-			return this.ratingArray;
-		},
+
+		return this.ratingArray;
+	},
+
+	displayErrorMessage(message) {
```
Comment (marcusmoller97) [inline challenges.js:31]: Maybe we can move displayErrorMessage function above getApiToArray for better readability and updates in the future.

**Review by marcusmoller97 — COMMENTED** [review]
It looks good, have a minor change that concerns the readability of the code. Nothing major but maybe something we should think about. Otherwise great work! 👍

**Comment by jennymakki** [comment]
Well done! Maybe we can style the text a bit different so that it is in one row and slightly bigger?

**Comment by Jorlindstrom** [comment]
I think we have to do som styling on this text. But the code is functional

**Comment by Tobias-hubs** [comment]
Thanks for feedback, fixed this in a new commit

**Pull request merged**